src/Components/CustomButton/CustomButton.tsx

```diff
@@ -1,7 +1,8 @@
 import React from "react";
 import { css, StyleDeclaration, StyleSheet } from "aphrodite";
 import Icon, { IconTypes } from "components/Icon/Icon";
-
+import { typography } from "styles/typography";
+import { palette } from "styles/palette";
```

Comment:
Create src/styles/index.ts and import there typography and palette, then you can make one import -> {typography, palette} from 'styles'